### DISCUSSION TITLE
官方网站首页的技术栈里面有写基于Shadcn UI，现在是弃用了吗？

### DISCUSSION THREAD

**Discussion by codenamea** [vbenjs/vue-vben-admin#4475] [General]
我下载了项目回来发现默认的UI库是只能选ant design , element , naive 呢？
请问是我理解问题还是，已经弃用了？

**Comment by yc0123450** [comment]
不是弃用，整体框架的UI用的是Shadcn，比如导航布局这些，这样就可以兼容任何第三方UI库
